src/parser/shared/modules/items/bfa/dungeons/RotcrustedVoodooDoll.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS/index';
+import ITEMS from 'common/ITEMS/index';
+import Analyzer from 'parser/core/Analyzer';
+import { formatNumber } from 'common/format';
+import ItemDamageDone from 'interface/others/ItemDamageDone';
+import Abilities from 'parser/shared/modules/Abilities';
+
+/**
+ * Rotcrusted Voodoo Doll
+ * Use: Brandish the Voodoo Doll at your target, 
+ * dealing X Shadow damage over 6 sec, and an additional X Shadow damage after 6 sec. (2 Min Cooldown)
+ */
+class RotcrustedVoodooDoll extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+  };
+
+  damage = 0;
+  ticks = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrinket(ITEMS.ROTCRUSTED_VOODOO_DOLL.id);
+
+    if (this.active) {
+        this.abilities.add({
+          spell: SPELLS.ROTCRUSTED_VOODOO_DOLL_BUFF,
+          buffSpellId: SPELLS.ROTCRUSTED_VOODOO_DOLL_TICK.id,
+          name: ITEMS.ROTCRUSTED_VOODOO_DOLL.name,
+          category: Abilities.SPELL_CATEGORIES.ITEMS,
+          cooldown: 120,
```

Comment:
No magic numbers, please pull this out as a constant and use a variable.

Reply:
I think the usage is fine here, we might as well make dozens of cooldown constants in regular Abilities.js in that case 😄

Reply:
So how do we go on about this? I think having Abilities.js the way it is is ok, but specific modules should probably take care of the rule?